ME Service Desk Plus API fix (#9579)
demisto/content: Packs/ServiceDeskPlus/ReleaseNotes/1_2_1.md

#### Integrations
##### Service Desk Plus

> - Added a minor API fix. Close and assign request format changed from `api/v3/requests/<Request Id>/close` and `api/v3/requests/<Request Id>/assign` to `api/v3/requests/<Request Id>/_close` and `api/v3/requests/<Request Id>/_assign` respectively. 
> - This change might break BC if not updating the pack as the previous format will be deprecated by November 15th 2020.

Reviewer: ```suggestion
- Updeted API endpoints of assign and close because the previous are planed to be deprecated in November 15th 2020.
```
